test/run

```diff
@@ -4,5 +4,12 @@
 
 set -eu
 
+# run static code checks like pyflakes and pep8
+PYEXEFILES="$(git grep -lI '^#!.*python') $(git ls-files "*.py")"
+python3 -m pyflakes $PYEXEFILES
+
+# TODO: Fix long lines
+python3 -m pycodestyle --ignore E501 $PYEXEFILES
```

Comment:
Hint: Start with `--max-line-length=<longest line that we currently have`, and work your way down to ~ 130 or so. That seems to be our current practice (80 is really a bit low IMHO)

Reply:
Updated, our recordman is 194 though :)